projects/kubernetes/kubernetes/Makefile

```diff
@@ -96,10 +95,6 @@ test: $(BINARY_TARGETS)
 fix-licenses:
 	build/fix_licenses.sh $(REPO) $(OUTPUT_DIR)
 
-.PHONY: create-version-file
```

Comment:
We still need to make the file because the release tooling uses it.

Reply:
So we need to make it but not have it in the repo?

Reply:
correct